pkg/services/app/install.go

```diff
@@ -0,0 +1,9 @@
+package app
+
+import "fmt"
+
+type InstallParams struct{}
+
+func (a *App) Install(params InstallParams) error {
```

Comment:
I sometimes get file fatigue with all my editor tabs. I would propose we keep these in the same file since they share module scope.

Reply:
sorry I didn't get what you mean.

Reply:
I would like to combine everything in `pkg/services/app/` into a single file (except for tests)

Reply:
ahh, if you open the entire file you'll see is huge, and that's the implementation of a single action. I'm afraid as we add more and more code will be super hard to navigate that.

Reply:
Now I am confused. This file (and the others like it) appear to be only 40 lines

Reply:
Nevermind I think I understand now